runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/fn/control/MeanByteCountMonitoringInfoToCounterUpdateTransformerTest.java

```diff
@@ -107,12 +108,12 @@ public void testTransformReturnsValidCounterUpdateWhenValidMonitoringInfoReceive
 
     CounterUpdate result = testObject.transform(monitoringInfo);
 
-    assertNotEquals(null, result);
-    assertEquals(
-        "{cumulative=true, integerMean={count={highBits=0, lowBits=0}, "
-            + "sum={highBits=0, lowBits=0}}, "
-            + "nameAndKind={kind=MEAN, "
-            + "name=transformedValue-MeanByteCount}}",
-        result.toString());
```

Comment:
This assertion was based on AbstractMap's `toString()`. This style is prone to unnecessary maintenance of the test code when upgrading dependencies. Dependencies may change the internal ordering of fields and trivial change in `toString()`. In fact, [BEAM-8695](https://issues.apache.org/jira/browse/BEAM-8695) is suspended partly because of these tests.